### PR TITLE
i18n(fr) Update and Translate `render.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### What kind of changes does this PR include?
- Updated and Translated content

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/render.mdx [hunk 1]
```diff
@@ -5,14 +5,15 @@ type: deploy
...
-1. Créer un [compte render.com](https://dashboard.render.com/) et se connecter
-2. Cliquer sur le bouton **Nouveau +** du tableau de bord et sélectionner **Site Statique**
-3. Connecter votre dépôt [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou, alternativement saisir l'adresse publique d'un dépôt publique
-4. Donner à votre site web un nom, sélectionner la branche et spécifier la commande de génération et le dossier de publication
-   - **commande de génération:** `npm run build`
-   - **dossier de publication:** `dist`
-5. Cliquer sur le bouton **Créer un Site Statique**
+1. Créer un [compte render.com](https://dashboard.render.com/) et connectez-vous
+2. Cliquer sur le bouton **New +** du tableau de bord et sélectionner **Static Site**
+3.Connectez votre dépôt [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou entrez l'URL publique d'un dépôt public.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/render.mdx:14]:
```suggestion
3. Connectez votre dépôt [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou entrez l'URL publique d'un dépôt public.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Super b @McFlyPartages, voici un mineur suggestion.

File: src/content/docs/fr/guides/deploy/render.mdx [hunk 2]
```diff
@@ -5,14 +5,15 @@ type: deploy
...
+1. Créer un [compte render.com](https://dashboard.render.com/) et connectez-vous
+2. Cliquer sur le bouton **New +** du tableau de bord et sélectionner **Static Site**
+3. Connectez votre dépôt [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou entrez l'URL publique d'un dépôt public.
+4. Donnez un nom à votre site web, sélectionnez la branche et spécifiez la commande de construction et le répertoire de publication
+   - **build command:** `npm run build` (Commande de construction)
+   - **publish directory:** `dist` (Répertoire de publication)
+   - **Environment variables (advanced)**: (Variables d'environnement) Par défaut, Render utilise Node.js 14.17.0, mais Astro [nécessite une version supérieure](/fr/install/auto/#prérequis).  Ajoutez une variable d'environnement avec une **Variable key** (clé de variable) de `NODE_VERSION` et une **Value** (Valeur) de `18.14.1` ou plus pour dire à Render d'utiliser une version compatible de Node.js. Alternativement, ajoutez un fichier [`.node-version`](https://render.com/docs/node-version) ou [`.nvmrc`](https://render.com/docs/node-version) file to your project to specify a Node.js version.
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/deploy/render.mdx:18]: `file to your project to specify a Node.js version.`
Hello ! Il manque de la traduction par ici!

**Review by thomasbnt — CHANGES_REQUESTED** [review]

**Comment by McFlyPartages** [comment]
LGTM

**Review by thomasbnt — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work 🙌

**Pull request merged**